load-data-to-bigquery/Ballerina.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+org = "lakshanweerasinghe"
```

Comment:
shall we add a `org` name like `ballerina`.

Reply:
use `etl`